test cases/windows/18 msvc charset/iso-8859-1.c

```diff
@@ -0,0 +1,7 @@
+#include <stdio.h>
+
+int main(int argc, char *argcv[])
+{
+  printf("This is ISO-8859-1 encoded �\n");
```

Comment:
Can we also have a unit test that this file fails to compile with `/utf-8`? Just in case something changes in the toolchain in the future that breaks everything.

Reply:
ok, added.